lib/atomic_admin/jwt_token.rb

```diff
@@ -0,0 +1,66 @@
+module AtomicAdmin
+  module JwtToken
```

Comment:
Is this the code that was copied from the rails apps? If so, should we call that out here?